src/utils/Firebase/RetrieveVideos.js

```diff
@@ -0,0 +1,17 @@
+export default function RetrieveVideos (dataFromFirebase, setVideoValue) {
```

Comment:
A better readability practice is to write the failure case in the `if` block (Because failure cases are smaller blocks of logic), this ensures that the meaty success case is not indented 

eg.
```javascript
export default function RetrieveVideos(dataFromFirebase, setVideoValue) {
	if (!Array.isArray(dataFromFirebase)) {
		console.error('dataFromFirebase is not an array');
		return;
	}
	const transformedData = dataFromFirebase.map((item) => ({
		category: item.category,
		messages: item.messages,
		operating_system: item.operating_system,
		stopTimes: item.stopTimes,
		tags: item.tags,
		url: item.url,
		subtopic: item.subtopic,
		key: item.key,
	}));
	setVideoValue(transformedData);
}
```